### ISSUE TITLE
A11y: screen readers don't read selected value and options

### ISSUE THREAD

**Issue by Fedoskina** [rob-balfre/svelte-select#286]
The select doesn't have aria-attributes. Because of that, during the keyboard navigation, the screen readers read the selected option as `some text edit blank` and all the options as blank too.

One of the simple solutions for the selected value could be:

```
<input
    readOnly={!isSearchable}
    {..._inputAttributes}
    bind:this={input}
    on:focus={handleFocus}
    bind:value={filterText | value}
    placeholder={placeholderText}
    style={inputStyles}
    disabled={isDisabled} />
```

**Comment by rob-balfre** [comment]
@Fedoskina I had a go at adding A11y support using react-select as a guide... 

Try out `4.2.8-beta.0` to see the changes. Thanks

**Comment by Fedoskina** [comment]
@rob-balfre, thank you for the update. I was trying it out. But I think it missed a key (magic) thing which react-select use:
1. span aria-selection -- for the selected item
2. span aria-context -- for options

![image](https://user-images.githubusercontent.com/5310513/127487023-51e300d3-e89e-4ffc-9901-b10d2e109ba3.png)

There is a bit of complex value [github link](https://github.com/JedWatson/react-select/blob/9009a153c059474117a7696f9f7a0914ad80acc0/packages/react-select/src/components/LiveRegion.tsx#L187-L188)

The reader reads spans when the value inside changes. 

Let me know if I can help in any way.

**Comment by rob-balfre** [comment]
@Fedoskina thanks, nice catch.  Added to `svelte-select@4.2.8-beta.1`

**Comment by Fedoskina** [comment]
@rob-balfre yay, it works super nice. Thanks a lot!!! 
Small suggestion: make a text `Use up and down to choose options, press enter to select the currently focused option, press escape to exit the menu, press tab to select the option and exit the menu.` is optional, coming from props.  What do you think?

**Comment by rob-balfre** [comment]
@Fedoskina added 3 exported methods for controlling A11y.

```js
export let ariaValues = (values) => {
  return `Option ${values}, selected.`;
}

export let ariaListOpen = (label, count) => {
  return `You are currently focused on option ${label}. There are ${count} results available.`;
}

export let ariaFocused = () => {
  return `Select is focused, type to refine list, press down to open the menu.`;
}
```

Changes available in `svelte-select@4.2.8-beta.2`.

**Comment by Fedoskina** [comment]
@rob-balfre, please add new props in type https://github.com/rob-balfre/svelte-select/blob/master/src/index.d.ts 

Apart from that works very well, thanks a lot for all your help

**Comment by rob-balfre** [comment]
@Fedoskina thanks 👍

Changes available in `svelte-select@4.3.0`
